### DISCUSSION TITLE
Will "file manager" automatically redirect to the route in console?

### DISCUSSION THREAD

**Discussion by requireCool** [kingToolbox/WindTerm#538] [Ideas]
When you change the current route/directory in console,  it could be better for file manager to follow the path change, and show the files in current directory.

**Comment by kingToolbox** [comment]
Sorry for the late reply.

This feature has been requested many times and will be implemented in version `WindTerm_2.4.0` or `WindTerm_2.5.0`. Please stay tuned, thank you.

**Reply by requireCool** [reply]
OK, We'll wait for that. Thanks for your contribution ;)
